src/main/webapp/app/app.component.ts

```diff
@@ -27,4 +29,8 @@ export class AppComponent implements OnInit {
   deleted(): void {
     this.tasks$ = this.taskService.getAll();
   }
+
+  searchTask(text: string): void {
+    this.searchedText = text;
```

Comment:
What made you choose this over a direct service call?

Reply:
If I subscribe on some Subject with using service in this component, I think I should have used manually change detection to update the list, here my Input is changed every time and components were pretty close one to another